.github/workflows/comment-pr-on-check-failure.yml

```diff
@@ -35,34 +36,32 @@ jobs:
             var fs = require('fs');
             fs.writeFileSync('${{github.workspace}}/pr.zip', Buffer.from(download.data));
       - run: unzip pr.zip
-      - name: DCO comment on PR
+      - name: DCO comment message
         if: ${{ github.event.workflow.name == 'DCO check' }}
-        uses: actions/github-script@v3
-        with:
-          script: |
-            var fs = require('fs');
-            var issue_number = Number(fs.readFileSync('./NR'));
-            var msg = `Oops! Looks like you failed the \`DCO check\`. Be sure to sign all your commits.
-            ### Howto
-            - [Magma guidelines on signing commits](https://magma.github.io/magma/docs/next/contributing/contribute_workflow#guidelines)
-            - [About the \`signoff\` feature](https://stackoverflow.com/questions/1962094/what-is-the-sign-off-feature-in-git-for)
-            - [Howto: sign-off most-recent commit](https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits)
-            - [Howto: sign-off multiple past commits](https://gist.github.com/kwk/d70f20d17b18c4f3296d)`
-            github.issues.createComment({
-              issue_number: issue_number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: msg,
-            })
-      - name: Python format comment on PR
+        run: |
+          echo "MSG=Oops! Looks like you failed the \`DCO check\`. Be sure to sign all your commits.
+          ### Howto
+          - [Magma guidelines on signing commits](https://magma.github.io/magma/docs/next/contributing/contribute_workflow#guidelines)
+          - [About the \`signoff\` feature](https://stackoverflow.com/questions/1962094/what-is-the-sign-off-feature-in-git-for)
+          - [Howto: sign-off most-recent commit](https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits)
+          - [Howto: sign-off multiple past commits](https://gist.github.com/kwk/d70f20d17b18c4f3296d)" >> $GITHUB_ENV
+      - name: Python format comment message
         if: ${{ github.event.workflow.name == 'Python Format Check' }}
+        run: |
+          echo "MSG=Oops! Looks like you failed the \`Python Format Check\`.
+          Instructions on running the formatter locally are provided [here](https://github.com/magma/magma/tree/master/lte/gateway/docker/python-precommit)." >> $GITHUB_ENV
+      - name: Semantic PR comment message
+        if: ${{ github.event.workflow.name == 'Semantic PR' }}
```

Comment:
So for my understanding: does this only work if the check is a GH Actions check, or could we leave the check as the existing probot check and still test its outcome?

Reply:
yes we could let the bot lives if we want. I would have to revert the deletion of the config file tough

Reply:
Gotcha. The one nice thing about the existing bot is it updates ~immediately when you change the PR title, which makes it easy for me to update PR titles for contributors and have the check immediately pass

See my below comment about the `validateSingleCommit` option -- can you open a PR against the existing bot and check whether it handles this correctly? I think that will be the deciding factor on whether we should stay with probot or move to this GH Action

Reply:
The edit trigger will do the same with this github action workflow, it will recheck on a title change just like the bot.
And for the one commit PR, the pr bot passes as it only checks the title but when merging, github does take the commit message as merge commit title.
For reference: the PR https://github.com/quentinDERORY/magma/pull/16 and what it looks like in master (my repo lol) 
![image](https://user-images.githubusercontent.com/15911421/126514315-9c42d3ce-50de-463d-bc81-b50763caa8bc.png)